### ISSUE TITLE
New release proposal

### ISSUE THREAD

**Issue by WilliamTambellini** [docopt/docopt.cpp#129]
The last (and only) release was in 2016. Any new official release possible ? kind

**Comment by jaredgrubb** [comment]
There havent been too many changes since then, but happy to re-tag and update if that's helpful!

**Comment by Kerndog73** [comment]
@jaredgrubb Perhaps you could review [some PRs](https://github.com/docopt/docopt.cpp/pull/126) 😉. Breaking changes might be acceptable after a release...

**Comment by rdeterre** [comment]
Yes, it would be very useful to tag a new release now that the build works in Visual Studio 2019. I'd be happy to make a PR for it if you'd like.

**Comment by MiguelRocha** [comment]
Please consider doing a new release. The last one does not have pkg-config support.

**Comment by chrisdembia** [comment]
I woud love a new release, so that we could update the conda package. I depend on docopt.cpp, and would like to use its conda package, but the conda package is using the last release.

**Comment by SMillerDev** [comment]
@jaredgrubb since you're the last committer: any chance at a new release?

**Comment by jaredgrubb** [comment]
There are plenty of tags. What do you mean by "release"?

**Comment by SMillerDev** [comment]
The last tag/release was 6 years ago. It would be nice to have a new tag as a stable point to use in package managers or other software.
